improver_tests/calibration/ensemble_calibration/test_EstimateCoefficientsForEnsembleCalibration.py

```diff
@@ -352,6 +356,46 @@ def test_coefficients_from_mean(self):
 
         self.assertEqual([cube.name() for cube in result], self.expected_coeff_names)
 
+    @ManageWarnings(ignored_messages=IGNORED_MESSAGES, warning_types=WARNING_TYPES)
+    def test_coordinates_from_mean_period_diagnostic(self):
+        """Test that the time coordinates are as expected when the historic
+        forecasts are time-bounded diagnostics, e.g. maximum in hour."""
+
+        fp_bounds = [10800, 14400]
+
+        self.historic_forecast.coord("forecast_period").bounds = fp_bounds
+        expected_fp = self.expected_fp
```

Comment:
I don't get the point of this line. Why not just use self.expected_fp on the two occasions this new variable is used below?

Reply:
There is no point to that line, you're quite right. Changed.